Add user consent documentation (#352)
SRGSSR/pillarbox-android: pillarbox-analytics/docs/README.md

}
```

### Handle user consent
User consent can be configured at initialization:

```kotlin
val initialUserConsent = UserConsent(
    comScore = ComScoreUserConsent.UNKNOWN,
    commandersActConsentServices = emptyList()
    )

val config = AnalyticsConfig(
    vendor = AnalyticsConfig.Vendor.SRG,
    nonLocalizedApplicationName = "PillarboxDemo",
    appSiteName = "pillarbox-demo-android",
    sourceKey = AnalyticsConfig.SOURCE_KEY_SRG_DEBUG,
    userConsent = initialUserConsent
)

> initSRGAnalytics(config = config)

Author: No because `initSRGAnalytics ` is an extension of `Application`. An other way to initialize, is to call `SRGAnalytics.init(application = this, config = config)`.

```
Update user consent at runtime:
```kotlin
val updatedUserConsent = UserConsent(
    comScore = ComScoreUserConsent.DECLINED, //or ComScoreUserConsent.ACCEPTED
    commandersActConsentServices = listOf("service1_id", "service2_id")
    )
SRGAnalytics.setUserConsent(updatedUserConsent)
```

> User consent values are updated at the next analytics event.

Reviewer: ```suggestion
User consent values will be updated with the next analytics event.
```
